From a0b676ff4d8af6cfb1738b766ed3e231f069a424 Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Wed, 1 Jun 2022 16:14:16 +0800
Subject: [PATCH 23/31] Revert "drm/bridge: cdns-dsi: Make sure to to create
 proper aliases for dt"

This reverts commit 54bc98a0ab2b230907dd764bd8764068eb9c0c06.
---
 drivers/gpu/drm/bridge/cdns-dsi.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/drivers/gpu/drm/bridge/cdns-dsi.c b/drivers/gpu/drm/bridge/cdns-dsi.c
index 9c3f9110895e..0cb9dd6986ec 100644
--- a/drivers/gpu/drm/bridge/cdns-dsi.c
+++ b/drivers/gpu/drm/bridge/cdns-dsi.c
@@ -1284,7 +1284,6 @@ static const struct of_device_id cdns_dsi_of_match[] = {
 	{ .compatible = "cdns,dsi" },
 	{ },
 };
-MODULE_DEVICE_TABLE(of, cdns_dsi_of_match);
 
 static struct platform_driver cdns_dsi_platform_driver = {
 	.probe  = cdns_dsi_drm_probe,
-- 
2.27.0

